20240329_CHWIDE_component_separated/README.md

```diff
@@ -0,0 +1,38 @@
+# Documentation for CHWIDE component separated maps
+
+#### Data description
+We provide foreground cleaned CMB maps obtained with NILC for CMB-S4 Chile wide survey. There are separate maps for E and B-mode polarizations. 
+Currently, about 25 realizations are provided for testing purposes. The masks used for the E and B maps are also provided. All maps are at `NSIDE=2048` 
+and `lmax=4096`, with 2.1 arcmin gaussian beam. The current maps are only with the PySM medium complexity foreground model.
+
+#### Input simulations and NILC pipeline
+The input maps are simulated with the following components:
+* Input foregrounds are combined medium complexity foregrounds from `combined_foregrounds_mediumcomplexity`.
```

Comment:
What is an "Input foreground"? All foregrounds should already be in the medium complexity foregrounds.

Reply:
Input foregrounds mean foreground maps that are used in simulations. I rephrased the sentence. See if it makes more sense now.